src/pages/requests/index.tsx

```diff
@@ -183,7 +186,9 @@ const RequestsPage: FC<Props> = ({
           </SC.Text>
           <Button
             onClick={() => {
-              window.open(`${FDK_COMMUNITY_BASE_URI}/category/6`);
+              window.open(
+                `${FDK_COMMUNITY_BASE_URI}/topic/56/etterspørr-data-api`
```

Comment:
Definer gjerne i variabel hva `56` er så dokumenterer koden seg selv.
`const variabelNavnSomForklarerMagiskTall = "56"` og så interpoler i URL-en.
```
const requestTopicNumber = "56" // Er det "requestTopicNumber" eller annet?
${FDK_COMMUNITY_BASE_URI}/topic/${requestTopicNumber}/etterspørr-data-api
```

Reply:
Tenkte kanskje det var tydelig nok i urlen - topic/56/etterspørr-data-api -> man kommer til topic 56 "Etterspør data api" ?

Reply:
Men hva er topic 56? Var ikke tydelig for meg at "Etterspør-data-api" og topic 56 det samme.